.github/workflows/label-new-issues.yaml

```diff
@@ -0,0 +1,19 @@
+# This workflow is used to add the 'triage-needed' label to newly opened issues.
+
+name: Label new issues
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+jobs:
+  label_issues:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Label issues
+        uses: andymckay/labeler@5c59dabdfd4dd5bd9c6e6d255b01b9d764af4414
+        with:
+          add-labels: "triage-needed"
```

Comment:
I see triage added to new issues, not triage-needed. Bug?

Reply:
No it's just that when I tested on my fork, I used the `triage` label. But I later saw that this repository already had the `triage-needed` label. So I thought it'd be better to use it instead of `triage`. I just didn't run the test again on my fork.